src/modality-specific-files/magnetic-resonance-spectroscopy.md

```diff
@@ -0,0 +1,191 @@
+
+Each `<label>` in the table MAY be combined with another to better describe the acquisition used.
+For example, `megaspecial`, `jpress`, `dwslaser`, and so on.
+
+The OPTIONAL `nuc-<label>` key/value pair can be used to distinguish acquisitions tuned to detect different nuclei.
+The label is the name of the nucleus or nuclei, which corresponds to DICOM Tag 0018, 9100.
+For example, `nuc-1H`, `nuc-31P`, `nuc-1H13C`.
+The key `ResonantNucleus` MUST also be included in the sidecar JSON file, with the same label.
+
+Similarly, the OPTIONAL `voi-<label>` key/value pair can be used to distinguish between repeated
+acquisitions localized to different regions (that is, acquisitions with different VOI).
+The label SHOULD be the name of the body region or part scanned.
+If used, the entities `BodyPart` and `BodyPartDetails` MUST also be included in the sidecar JSON file.
+
+## Sidecar JSON
+
+MRS data files MUST be described by metadata fields, stored in sidecar JSON files (`*.json`).
+
+### Common metadata fields
+
+Metadata described in the following sections are shared with other MR modalities and SHOULD be
+present in the sidecar JSON files.
+
+#### Scanner hardware
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+
+{{ MACROS___make_sidecar_table("mrs.MRSScannerHardware") }}
+
+#### Sequence specifics
+
+{{ MACROS___make_sidecar_table("mrs.MRSSequenceSpecifics") }}
+
+### MRS-specific fields
+
+Metadata fields that MUST be present:
+
+{{ MACROS___make_sidecar_table("mrs.MRSRequiredFields") }}
+
+SHOULD be present:
+
+{{ MACROS___make_sidecar_table("mrs.MRSRecommendedFields") }}
+
+MAY be present:
+
+{{ MACROS___make_sidecar_table("mrs.MRSOptionalFields") }}
+
+### Example `*_svs.json`
+
+```JSON
+{
+   "InstitutionName": "Weill Cornell Medicine",
+   "InstitutionAddress": "1300 York Avenue, New York, NY 10065, USA",
+   "Manufacturer": "GE",
+   "ManufacturersModelName": "Discovery MR750",
+   "MagneticFieldStrength": 3,
+   "PulseSequenceType": "PRESS",
+   "ResonantNucleus": "1H",
+   "SpectrometerFrequency": 127.771,
+   "SpectralWidth": 2000,
+   "EchoTime": 0.035,
+   "NumberOfSpectralPoints": 2048,
+   "NumberOfTransients": 64,
+   "RepetitionTime": 2,
+   "AcquisitionVoxelSize": [40, 20, 30],
+   "BodyPart": "Brain",
+   "BodyPartDetails": "Anterior cingulate cortex",
+   "ReferenceSignal": "bids::sub-01/mrs/sub-01_acq-press_ref.nii.gz",
+   "AnatomicalImage": "bids::sub-01/anat/sub-01_T1w.nii.gz"
+}
+```
```

Comment:
```suggestion
```
### Combining MRS with anatomical MRI data
For combining MRS with anatomical MRI data, see [MRS-MRI correspondence](https://bids-specification.readthedocs.io/en/stable/appendices/cross-modality-correspondence.html#mrs-mri-correspondence) on the cross modality correspondence section of the Appendix.
```

Reply:
Added this just now ([873fd79](https://github.com/bids-standard/bids-specification/pull/1377/commits/873fd79846278d2218d56ee6e668312f8262b2aa))